python/cudf_kafka/cudf_kafka/_lib/kafka.pxd

```diff
@@ -65,7 +65,7 @@ cdef class KafkaDatasource(Datasource):
 
     cpdef int64_t get_committed_offset(self, string topic, int32_t partition)
 
-    cpdef map[string, vector[int32_t]] list_topics(self)
+    cpdef map[string, vector[int32_t]] list_topics(self, string tp) except +*
```

Comment:
Not sure why Cython is crashing, but tested some similar locally and this seems to work for me:

```suggestion
    cpdef map[string, vector[int32_t]] list_topics(self, string tp) except *
```

Also needs a similar change in `kafka.pyx`